pom.xml

```diff
@@ -177,6 +178,13 @@
           <groupId>org.codehaus.mojo</groupId>
           <artifactId>animal-sniffer-maven-plugin</artifactId>
           <version>1.23</version>
+          <dependencies>
+            <dependency>
+              <groupId>org.ow2.asm</groupId>
+              <artifactId>asm</artifactId>
+              <version>9.6</version>
+            </dependency>
+          </dependencies>
```

Comment:
ASM will need to be upgraded to `9.6` transitively for `animal-sniffer`, in order to support Java 21 class bytecode. This change is inert for now, but avoids issues if Guava classes are built at higher bytecode levels.